jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerLoadManifestTemplate.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
The convention currently for JSON templates in this project is to put them under a `json` package.